phoneApp/keypad.js

```diff
@@ -0,0 +1,91 @@
+class Keypad {
+	constructor() {
+		this.buttonsContent = ['1', '2', '3', '4', '5', '6', '7', '8', '9', '*', '0', '#'];
```

Comment:
We should create numbers as static content, no need to store them in any structures.

How to understand what we should store in a variable and what is just static?

If such data sent from the server you have to store it scenarios (around 95% of all scenarios) it's just a static data.

Furthermore, there will never exist numbers like "11" on keypad at phone :)